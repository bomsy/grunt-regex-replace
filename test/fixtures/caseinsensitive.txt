Expect [field], [FielD], [FIELD] caseinsensitive.txt