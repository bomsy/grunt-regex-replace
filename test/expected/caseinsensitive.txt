Expect [replaced], [replaced], [replaced] caseinsensitive.txt